src/openloco/objects/vehicle_object.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "../localisation/stringmgr.h"
+
+namespace openloco
+{
+    struct vehicle_object_unk
+    {
+        uint8_t length;         // 0x00
+        uint8_t pad_01[0x04 - 0x01];
+        uint8_t sprite_ind;     // 0x04
+        uint8_t var_05;
+    };
+
+    struct vehicle_object_sprite
+    {
+        uint8_t num_dir;        // 0x00
+        uint8_t pad_01[0x03 - 0x01];
+        uint8_t vehicle_type;   // 0x03
+        uint8_t num_units;      // 0x04
+        uint8_t pad_05;
+        uint8_t bogey_position; // 0x06
+        uint8_t flags;          // 0x07
+        uint8_t pad_08[0x0E - 0x08];
+        uint8_t sprite_num;     // 0x0E
+        uint8_t pad_0F[0x1E - 0xF];
+    };
+
+    struct vehicle_object
+    {
+        string_id name; // 0x00 probably not confirmed
+        uint8_t vehicle_class;  // 0x02
+        uint8_t type;           // 0x03
```

Comment:
type (controls in which tab the vehicle is when you want to buy it (0=trains,1=busses,2=trucks,3=trams,4=planes,5=ships))